src/server/system_services/stats_aggregator.js

```diff
@@ -518,7 +518,8 @@ async function _partial_buckets_info(req) {
                 capacity_precent: (is_capacity_relevant && bucket_total > 0) ? size_utils.bigint_to_json(bucket_used.multiply(100)
                     .divide(bucket_total)) : 0,
                 is_healthy: _.includes(OPTIMAL_MODES, bucket_info.mode),
-                tagging: bucket_info.tagging || []
+                tagging: bucket_info.tagging || [],
+                objectbucket_used_bytes: size_utils.bigint_to_json(bucket_used)
```

Comment:
as you probably noticed, `bucket_used` is of type `BigInteger`, which is a third-party class we are using.  
The `bigint_to_json` function is converting the BigInteger into types that we can store in the DB or serialize in the RPC. This function usually returns a primitive Number, but it can also return an object if the value is over 1 PetaBytes. In this case, the API validation will fail.  
I suggest returning the internal value of the BigInteger:

```suggestion
                objectbucket_used_bytes: bucket_used.value
```

This will return a primitive Number up to `Number.MAX_SAFE_INTEGER` which is ~7 PetaBytes

Reply:
Ya that makes sense will update the code